serialization.md

```diff
@@ -327,7 +327,6 @@ Note: whitespace has been added to this example for clarity. Whitespace is OPTIO
         <ul>
           <li>
             <code>type</code> is usually set to <code>layers</code>.
-	    There is also a Windows-specific value <code>layers+base</code> that allows a base layer to be specified in a field of <code>rootfs</code> called <code>base_layer</code>.
```

Comment:
On Mon, Aug 29, 2016 at 04:31:23PM -0700, Stephen Day wrote:
> > If the intention here is to only allow the layers-type rootfses, I
> > don't see a problem with dropping the field from the spec.
>
> I'm not sure that you have full knowledge to evaluate the usage of
> this field, nor is this PR the forum for that discussion.

I'm sure you're right and I don't have sufficient background to
evaluate the usage of this field.  I would expect that this spec
(since it defines the field) would provide sufficient background on
how it is expected to be used, and what it means is the value is
‘layers’, or unset, or some other string.  But if that discussion is
supposed to happen somewhere else, I'm happy to have the discussion
there.  Can you point me at the appropriate forum?